### ISSUE TITLE
post_train_problem

### ISSUE THREAD

**Issue by AAwilliam** [nvidia-cosmos/cosmos-transfer2.5#57]
When I use the training script you provided for training, the loss occasionally becomes zero. What could be the reason?
Why is the training speed with multi-machine multi-GPU setup the same as that with a single machine (8 GPUs)?

<img width="1177" height="38" alt="Image" src="https://github.com/user-attachments/assets/d8ece222-cd14-482a-ac42-6242ba9dc5fd" />

<img width="1180" height="37" alt="Image" src="https://github.com/user-attachments/assets/6969c641-cb8b-48cc-ad09-da5acd729ad6" />

**Comment by pjannaty** [comment]
@YananJian could you please take a stab at finetuning and preparing instructions?

**Comment by pjannaty** [comment]
Instructions updated. Please consult https://github.com/nvidia-cosmos/cosmos-transfer2.5/blob/main/docs/post-training.md.
